modules/comment/comment.service.js

```diff
@@ -49,6 +52,18 @@ class CommentsService extends FilterHelper {
     return comment;
   }
 
+  async getRecentComments(limit) {
+    try {
+      const comments = await Comment.find()
+        .sort({ date: -1 })
+        .limit(limit)
+        .exec();
+      return comments;
+    } catch (error) {
+      throw new RuleError(error.message, error.statusCode);
+    }
```

Comment:
you need do something like that...without try..catch here
![image](https://user-images.githubusercontent.com/48614659/111352601-03f79500-868d-11eb-8fdf-5a42e3360c22.png)